.github/workflows/build.yaml

```diff
@@ -0,0 +1,25 @@
+name: Build
+on: [push, pull_request ]
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+
+    env:
+      DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+      DOTNET_CLI_TELEMETRY_OPTOUT: 1
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - run: |
+        pushd test/Tmds.LibC.Tests && dotnet test && popd
+        dotnet pack src/Tmds.LibC --configuration Release --version-suffix "$(date +"%y%m%d")-$TRAVIS_BUILD_NUMBER" --output .
+        
+    - uses: actions/upload-artifact@v2
```

Comment:
Can you remove this step?

Reply:
I removed this step, although I found it useful to be able to retrieve the NuGet packages from PR builds.